lib/marginalia/sidekiq_instrumentation.rb

```diff
@@ -13,7 +13,28 @@ def call(worker, msg, queue)
       end
     end
 
-    def self.enable!
+    def self.enable!(annotate_delayed_class_extension: false)
+      if annotate_delayed_class_extension
+        Sidekiq::Extensions::DelayedClass.class_eval do
+          # NOTE: Prefer redefining perform so that YAML is only parsed once.
+          # Adding a Marginalia::Comment.components would incur an extra parsing on every query
```

Comment:
Did you have an example of an alternative to what was done here, but using the Marginalia::Comment.components mechanism that's documented to show this as a custom component?

I'm hunting for custom component examples.